src/data/content/norme-e-riferimenti/linee-guida-di-design/requisito-4-1/azione-1.yaml

```diff
@@ -0,0 +1,357 @@
+          ctas:
+          - label: Vai all'argomento sul Manuale #C
+            screenReaderText: "Accessibilità"
+            url: "https://docs.italia.it/italia/designers-italia/manuale-operativo-design-docs/it/versione-corrente/doc/esperienza-utente/accessibilita.html#accessibilita" #M
+            blank: true #M #C true if new tab for external links
+            icon:
+              icon: sprites.svg#it-external-link #C #I #it-arrow-right | it-external-link for external links
+              color: primary
+              align: middle
+              hidden: true
+              size: sm
+              addonClasses: ms-2
+
+# ///////////////////
+  # HIGHLIGHT WITH CARDS
+  # ///////////////////
+  highlightCards:
+    id: HC1
+    title: Scopri le risorse correlate a questa azione  #C
+    # text: Lorem ipsum #C
+    col4: false
+    nospace: false
+    nopadtop: true
+
+    # CARDS
+    cards: # AUTO?
+    
+      # CARD CHILDREN ACTION
+      - title: "Accessibilità" #C
+        fullHeight: true
+        imgRatio: 16x9
+        #iconImg: /icons/kit_coprog_contenuti.svg #I 
+        iconImgAlt: Alt lorem ipsum #C
+        rounded: true
+        url: "/design-system/fondamenti/accessibilita/" #M
+        text: | #C
+          Progettare e sviluppare interfacce digitali significa decidere continuamente chi includere, o escludere, dall’esperienza di utilizzo e fruizione, a seconda delle proprie caratteristiche, conoscenze, capacità o condizioni di disabilità, temporanee o meno
+        tag:
+          label: Fondamento #C
+        share: #AUTO
+          btnId: shareButton2
+          button:
+            addonStyle: icon-only-drop
+            ariaLabel: Condividi
+            icon:
+              icon: sprites.svg#it-more-items
+              size: sm
+              color: secondary
+          list:
+            isShare: true
+
+      # CARD CHILDREN ACTION
+      - title: "Progettare con qualità: l’accessibilità al centro del design system del Paese"
+        headingLevel: 4
+        img: /images/community/articoli-di-approfondimento/2021-medium-zaia-processo.webp
+        alt: Le risorse per progettare di Designers Italia
+        imgRatio: 16x9
+        imgPlaceholder: false
+        fullHeight: true
+        rounded: true
+        url: https://medium.com/designers-italia/progettare-con-qualita-accessibilita-al-centro-del-design-system-del-paese-5e3599170099
+        blank: true
+        externalLink:
+          label: Leggi su medium
+          screenReaderText: " l'articolo completo"
+          icon:
+            icon: sprites.svg#it-external-link
+            size: xs
+        moreInfo: di Daniele Tabellini e Fabrizio Caccavello
+        chips:
+          - color: secondary
+            label: Processo
```

Comment:
@danielaiozzo this card has different tags on the current website (/community), please update it

Reply:
done, please check